doc/pr2_tutorials/planning/scripts/doc/move_group_python_interface_tutorial.rst

```diff
@@ -15,18 +15,26 @@ The entire code can be seen :codedir:`here in the moveit_pr2 github project<plan
 
 The launch file
 ^^^^^^^^^^^^^^^
-The entire launch file is `here
-<https://github.com/ros-planning/moveit_tutorials/tree/kinetic-devel/doc/pr2_tutorials/planning/launch/move_group_python_interface_tutorial.launch>`_
+The entire launch file is `here <https://github.com/ros-planning/moveit_tutorials/tree/kinetic-devel/doc/pr2_tutorials/planning/launch/move_group_python_interface_tutorial.launch>`_
 on github. All the code in this tutorial can be run from the
 moveit_tutorials package that you have as part of your MoveIt! setup.
 
 Running the code
 ^^^^^^^^^^^^^^^^
 
-Roslaunch the launch file to run the code directly from moveit_tutorials::
+Make sure your python file is executable (chmod +x scripts/move_group_python_interface_tutorial.py).
```

Comment:
My understanding is that when a shell runs a file its properties need to be set to executable for the shell to actually execute the file, together with the shebang line ( #!/usr/bin/env python ) causes the python file to be executed as a python script when it is run in the shell (How this interacts with rosrun, I do not know)

To be honest I do not know if it is strictly necessary but I followed this tutorial, which mentions the step: [http://wiki.ros.org/ROS/Tutorials/WritingPublisherSubscriber(python)]

I might try to see if it can go without tonight,